### PR TITLE
Get or cache a fiber doesn't need conf

### PR DESCRIPTION
**Pull request by gczsjdy**
## What changes were proposed in this pull request?

Modified corresponding `FiberCacheManager.get` and `Fiber.cache` functions' signature, removed `conf`.

## How was this patch tested?

Existing tests.

### CODE REVIEW

**Comment by gczsjdy** [comment]
After this removement, `cacheLoader` in `GuavaOapCache` can be eliminated.

**Comment by gczsjdy** [comment]
@linhong-intel @carsonwang

File: src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala [hunk 1]
```diff
@@ -23,18 +23,18 @@ import java.util.concurrent.atomic.AtomicLong
 import scala.collection.JavaConverters._
 
 import com.google.common.cache._
-import org.apache.hadoop.conf.Configuration
 
 import org.apache.spark.internal.Logging
 import org.apache.spark.util.Utils
 
+
```
Comment (linhong-intel) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: extra line.

File: src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitmapReaderV2.scala [hunk 2]
```diff
@@ -27,6 +26,7 @@ import org.apache.spark.sql.execution.datasources.oap.index.impl.IndexFileReader
 import org.apache.spark.sql.execution.datasources.oap.utils.{BitmapUtils, OapBitmapWrappedFiberCache}
 import org.apache.spark.sql.types.StructType
 
+
```
Comment (linhong-intel) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitmapReaderV2.scala]: extra line?

File: src/test/scala/org/apache/spark/sql/execution/datasources/oap/utils/OapBitmapWrappedFiberCacheSuite.scala [hunk 3]
```diff
@@ -17,18 +17,17 @@
 
 package org.apache.spark.sql.execution.datasources.oap.utils
 
-import java.io.{ByteArrayOutputStream, DataOutputStream, File, FileOutputStream}
+import java.io.{ByteArrayOutputStream, DataOutputStream, FileOutputStream}
 
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FSDataInputStream, Path}
-import org.roaringbitmap.RoaringBitmap
-
```
Comment (linhong-intel) [inline src/test/scala/org/apache/spark/sql/execution/datasources/oap/utils/OapBitmapWrappedFiberCacheSuite.scala]: don't remove this line

**Comment by gczsjdy** [comment]
Thanks @linhong-intel

**Pull request merged**